components/rendering/src/markdown.rs

```diff
@@ -242,6 +242,10 @@ pub fn markdown_to_html(content: &str, context: &RenderContext) -> Result<Render
 
                         Event::Start(Tag::Image(link_type, src, title))
                     }
+                    Event::Start(Tag::Link(link_type, link, title)) if link.is_empty() => {
+                        error = Some(Error::msg("There is a link that is missing a URL"));
```

Comment:
Can we add the link type/title to the error message?

Reply:
I actually had it with the title in the message initially but for some reason the `title` variable there is an empty string. I didn't have more time to look into it. My initial guess was that maybe the Markdown parser is not properly parsing a link without a URL.

Reply:
ah interesting, I'll see if I can reproduce it and open an issue on pulldown-cmark

Reply:
@Keats did you manage to reproduce it? Is it really an issue or did I misunderstand something?

Reply:
I haven't tried yet, I'll check when I have time

Reply:
Ah nevermind, `title` is `[I'm an inline-style link with title](https://www.google.com "Google's Homepage")` Google's Homepage. It looks like you don't get the link text in the event